src/drivers/dw/dma.c

```diff
@@ -827,6 +827,7 @@ static void dw_dma_verify_transfer(struct dma_chan_data *channel,
 #else
 		while (lli->ctrl_hi & DW_CTLH_DONE(1)) {
 			lli->ctrl_hi &= ~DW_CTLH_DONE(1);
+			dcache_writeback_region(lli, sizeof(struct dw_lli));
```

Comment:
The `lli` address is obtained by calling`platform_dw_dma_lli_get()`, which calls `cache_to_uncache()` on CAVS. Doesn't this guarantee, that we don't have to handle caches? Otherwise we'd also need to invalidate caches first?

Reply:
So this is maybe only needed for zephyr, possibly as part of early dev - let me see if it's still needed as Zephyr kernel uses uncache only but SOF uses both when using Zephyr.